src/Heading/Heading.tsx

```diff
@@ -21,24 +27,44 @@ export interface HeadingProps {
    * @type {string}
    * @description Sizes map to the available font-sizes from the defined list of font-types.
    * Adjust for visual hierarchy.
+   *
+   * TODO: Update this to only use the HeadingSizes keys and update the types to not use string
    */
-  size: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl' | 'xxxl';
+  size: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl' | 'xxxl' | 'large' | 'medium' | 'small';
   textAlign?: 'left' | 'center' | 'right';
   weight?: 'regular' | 'medium' | 'bold';
 }
 
-const LevelToSizeMap = {
-  1: 'xxxl',
-  2: 'xxl',
-  3: 'xl',
-  4: 'lg',
-  5: 'md',
-  6: 'sm',
+const SizeToClass = {
+  xs: styles.xs,
+  sm: styles.sm,
+  md: styles.md,
+  lg: styles.lg,
+  xl: styles.xl,
+  xxl: styles.xxl,
+  xxxl: styles.xxxl,
+  [HeadingSizes.LARGE]: styles.xxxl,
+  [HeadingSizes.MEDIUM]: styles.xl,
+  [HeadingSizes.SMALL]: styles.lg,
+};
+
+/* TODO Update this so that 2 is xl and 3 is lg */
+const LevelToClass = {
+  1: styles.xxxl,
+  2: styles.xxl,
+  3: styles.xl,
+  4: styles.lg,
+  5: styles.md,
+  6: styles.sm,
 };
 
-const getHeadingSizeClassFromLevel = (level: number) => `Heading--${LevelToSizeMap[level]}`;
+const WeightToClass = {
```

Comment:
💅 purely a nit but i would probably name this and the other lookup objects you have here to something like
```suggestion
const weightClasses = {
```

or 
```suggestion
const WEIGHT_CLASSES = {
```